documentation/CHANGELOG_TEMPLATE.md

```diff
@@ -25,12 +26,12 @@ This release contains the definition and documentation of
 <!--* API-name2 vx.y.z - unchanged-->
 <!--In case the repository (and hence its release) contains multiple APIs, list them all here. Also mention if an API in the repository is unchanged. Note: There shall be no "wip" API version in the repository at the time of release.-->
 
-<!--For any API version, if known, and for public API versions, include the references to the related Commonalities and ICM versions as follows:-->
+<!--For any API version, if known, and for public API versions, include the link to the related Commonalities and ICM releases:-->
```

Comment:
There is no link (in the sense of URL), so maybe:

```suggestion
<!--For any API version, if known, and for public API versions, include the Commonalities and ICM releases related to the API version (note: add the patch number only if the difference is relevant, otherwise refer only to the major and minor version:-->
```